packages/asc-core/src/components/Radio/RadioLabel.ts

```diff
@@ -1,24 +1,32 @@
 import styled from '../../styled-components'
+import getThemeColor from '../../utils/getThemeColor'
 
-interface Props {
+type Props = {
   htmlFor: string
   disabled?: boolean
 }
 
-const Label = styled.label.attrs<Props>({
-  htmlFor: ({ htmlFor }: Props) => htmlFor,
-  disabled: ({ disabled }: Props) => disabled,
-})<Props>`
-  color: ${props => (props.disabled ? '#bebebe' : '#434343')};
+const Label = styled.label.attrs<Props>(props => ({
+  htmlFor: props.htmlFor,
+  disabled: props.disabled,
+}))<Props>`
+  color: ${({ theme, disabled }) =>
+    disabled
+      ? getThemeColor(theme, 'tint', 'level4')
+      : getThemeColor(theme, 'tint', 'level5')};
   font-family: 'AvenirNextLTW01-Regular', verdana, sans-serif;
   font-size: 16px;
   padding-left: 38px;
   line-height: 28px;
 
   &::before {
     border-radius: 50%;
-    border: ${props =>
-      props.disabled ? '1px solid #bebebe' : '1px solid #000'};
+    border: ${({ theme, disabled }) =>
+      `1px solid ${
```

Comment:
As far as I know styled components, nested template literals should be used in the `css` function of the `styled-components` package. 
Like so:
```jsx
border: ${({ theme, disabled }) => css(`1px solid ${
  disabled
    ? getThemeColor(theme, 'tint', 'level5')
    : getThemeColor(theme, 'tint', 'level7')
}`)};
```

Reply:
> Or perhaps even:
> 
> ```sass
> border: 1px solid ${({ disabled }) => getThemeColor(theme, 'tint',  disabled ? 'level5' : 'level7')}
> ```

Yes, better